dashboard/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+
+import "./Tabs.scss";
+
+interface ITabsProps {
+  columns: string[];
+  data: Array<string | JSX.Element | JSX.Element[]>;
+}
+
+export default function Tabs({ columns, data }: ITabsProps) {
+  const [selected, setSelected] = useState(0);
+  const handleClick = (tab: number) => {
+    return () => setSelected(tab);
+  };
+  return (
+    <div className="tabs">
+      <ul className="nav" role="tablist">
+        {columns.map((column, index) => {
+          return (
+            <li role="presentation" className="nav-item" key={`${column}-${index}`}>
+              <button
+                id={`tab${index}`}
+                className={`btn btn-link nav-link tab-button ${selected === index ? "active" : ""}`}
+                aria-controls={`panel${index}`}
```

Comment:
To avoid having issues with duplicated `id` in the same view, I'd ask parent components to define a `prefix`/`id` for a `Tabs` component. In that way, you can create unique ids per `Tabs` component:

```
`${id}-content-${index}
```